themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -1640,6 +1648,12 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
             $scope.clientEdit.attributes["backchannel.logout.session.required"] = "false";
         }
 
+        if ($scope.backchannelUserCodeParameterRequired == true) {
+            $scope.clientEdit.attributes["backchannel.user.code.parameter.required"] = "true";
```

Comment:
Could you follow what I commented on `OIDCConfigAttributes`?

Reply:
Changed this, thank you.

Reply:
I've confirmed. Thank you.